tests/wp-unit/include/Core/Pages/Metabox/DeletePagesByStatusMetaboxTest.php

```diff
@@ -75,12 +98,142 @@ public function test_that_published_pages_can_be_deleted() {
 		);
 
 		$pages_deleted = $this->metabox->delete( $delete_options );
-		$this->assertEquals( 10, $pages_deleted );
+		$this->assertEquals( 20, $pages_deleted );
+
+		$published_pages = $this->get_pages_by_status();
+		$this->assertEquals( 0, count( $published_pages ) );
+
+		$trash_pages = $this->get_pages_by_status( 'trash' );
+		$this->assertEquals( 0, count( $trash_pages ) );
+	}
+
+	/**
+	 * Test date filter (older than x days) with a single post status.
```

Comment:
The comment says single post status, but inside you are creating two post status.